types/collection/cluster.go

```diff
@@ -25,6 +25,9 @@ import (
 // ClusterMetadataKind defines the kind of cluster metadata file
 const ClusterMetadataKind types.Kind = "ClusterMetadata"
 
+// QaLabelKey is the keyname for the qaLabel
+const QaLabelKey = "qalabel"
```

Comment:
we might want to use a more elaborate label like move2kube.konveyor.io/clusterqalabel